features/include/pcl/features/principal_curvatures.h

```diff
@@ -58,7 +58,7 @@ namespace pcl
     * \ingroup features
     */
   template <typename PointInT, typename PointNT, typename PointOutT = pcl::PrincipalCurvatures>
-  class PrincipalCurvaturesEstimation : public FeatureFromNormals<PointInT, PointNT, PointOutT>
+  class PCL_EXPORTS PrincipalCurvaturesEstimation : public FeatureFromNormals<PointInT, PointNT, PointOutT>
```

Comment:
Same here, why did you decide to add this? This macro is added during instantiation:

https://github.com/PointCloudLibrary/pcl/blob/8bc7c43b32fa497bd3f2b8c64b40e3b99ab12666/features/include/pcl/features/impl/principal_curvatures.hpp#L164